plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/BridgingHiveMetastore.java

```diff
@@ -132,8 +135,11 @@ public void updateTableStatistics(String databaseName, String tableName, AcidTra
     @Override
     public void updatePartitionStatistics(Table table, Map<String, Function<PartitionStatistics, PartitionStatistics>> updates)
     {
+        long startTime = System.currentTimeMillis();
         org.apache.hadoop.hive.metastore.api.Table metastoreTable = toMetastoreApiTable(table);
-        updates.forEach((partitionName, update) -> delegate.updatePartitionStatistics(metastoreTable, partitionName, update));
+        delegate.updatePartitionStatistics(metastoreTable, updates);
+        long endTime = System.currentTimeMillis();
```

Comment:
Is the measuring the duration of the `ANALYZE` command for debug logging relevant for production or is this a left-over?